bundles/org.openhab.binding.hapero/src/main/java/org/openhab/binding/hapero/internal/device/Device.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hapero.internal.device;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link Device} class contains the raw
+ * data received from the heating control system and
+ * conversion functions.
+ *
+ * @author Daniel Walter - Initial contribution
+ */
+@NonNullByDefault
+public class Device {
+    /** Raw data stream from the device */
+    protected String[] dataItems = {};
+    private final Logger logger = LoggerFactory.getLogger(Device.class);
```

Comment:
this line should go above `protected String[] dataItems = {};`

Reply:
OK, fixed